apps/arweave/src/ar_entropy_gen.erl

```diff
@@ -416,6 +416,16 @@ generate_entropies(RewardAddr, PaddedEndOffset) ->
 	end,
 	Entropies.
 
+shift_entropy_offset(BucketEndOffset, StoreID) ->
```

Comment:
```suggestion
advance_entropy_offset(BucketEndOffset, StoreID) ->
```

Maybe rename just to avoid confusion with the ar_entropy_storage:shift_entropy_offset?